solara/components/slider.py

```diff
@@ -21,7 +23,8 @@ def SliderInt(
     max: int = 10,
     step: int = 1,
     on_value: Optional[Callable[[int], None]] = None,
-    thumb_label=True,
+    thumb_label: Union[bool, Literal["always"]] = "always",
+    tick_labels: Union[List[str], Literal["end_points"], bool] = False,
```

Comment:
```suggestion
    tick_labels: Union[List[str], Literal["end_points"], bool] = False,
```

mypy complaints about this begin improper typed. Does vuetify treat "" and None differently? If not, we can do a `[""] * num_repeats` below?

Reply:
Vueitify treats them the same!

I'll update 👍